packages/api/tests/endpoints/collections/create-collection.js

```diff
@@ -348,7 +327,7 @@ test.serial('POST with non-matching file.regex returns 400 bad request repsonse'
 });
 
 test.serial('POST returns a 500 response if record creation throws unexpected error', async (t) => {
-  const stub = sinon.stub(Collection.prototype, 'create')
+  const stub = sinon.stub(CollectionPgModel.prototype, 'create')
```

Comment:
I'm curious what error message is returned from the endpoint if we inspect it in this test. I wouldn't expect this type of mocking for the test to work, but I might be missing something

Reply:
{"error":{"name":"Error","message":"### unexpected error","stack":["Error: unexpected error","    at CollectionPgModel.<anonymous> (/Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/packages/api/tests/endpoints/collections/create-collection.js:357:13)","    at Object.invoke (/Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/node_modules/sinon/lib/sinon/behavior.js:163:32)","    at CollectionPgModel.functionStub (/Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/node_modules/sinon/lib/sinon/stub.js:39:43)","    at Function.invoke (/Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/node_modules/sinon/lib/sinon/proxy-invoke.js:47:47)","    at CollectionPgModel.create (/Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/node_modules/sinon/lib/sinon/proxy.js:226:26)","    at /Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/packages/api/endpoints/collections.js:141:56","    at /Users/vpnguye2/Desktop/khuyen/EED/git/cumulus/packages/db/node_modules/knex/lib/execution/transaction.js:221:22","    at processTicksAndRejections (node:internal/process/task_queues:96:5)"]},### "level":"error","message":"Error occurred while trying to create collection: {\"name\":\"collectionName97fce3585b\",\"version\":\"0.0.0\",\"duplicateHandling\":\"replace\",\"granuleId\":\"^MOD09GQ\\\\.A[\\\\d]{7}.[\\\\S]{6}\\\\.006\\\\.[\\\\d]{13}$\",\"granuleIdExtraction\":\"(MOD09GQ\\\\.(.*))\\\\.hdf\",\"sampleFileName\":\"MOD09GQ.A2017025.h21v00.006.2017034065104.hdf\",\"files\":[],\"reportToEms\":true,\"createdAt\":1648492841920,\"updatedAt\":1648492841920}","sender":"@cumulus/api/collections","timestamp":"2022-03-28T18:40:41.942Z"}
[28/Mar/2022:18:40:41 +0000] "POST /collections HTTP/1.1"500 96 "-" "node-superagent/3.8.3" -